doc/features/currencies.rst

```diff
@@ -49,6 +49,35 @@ The Bitcoin currencies provides a single currency: the Bitcoin. It uses XBT as i
     $currencies->contains(new Currency('USD')); // returns boolean whether USD is available in this repository (false)
     $currencies->subunitFor(new Currency('XBT')); // returns the subunit for the Bitcoin (8)
 
+CurrencyList
+-------------
```

Comment:
Looks like there is one extra dash here. ;)

Reply:
Done.